i3-agenda.py

```diff
@@ -39,7 +43,9 @@ def default(self, object):
 def main():
     args = parser.parse_args()
 
-
+    if button is not "":
+        subprocess.Popen(["xdg-open", "https://calendar.google.com/calendar/r/day"])
```

Comment:
very cool, could you just move the URL to a const?

Reply:
Sure! Just did it

Reply:
cool thanks